typed-builder-macro/src/struct_info.rs

```diff
@@ -285,13 +287,76 @@ impl<'a> StructInfo<'a> {
                 #[deprecated(
                     note = #repeated_fields_error_message
                 )]
-                pub fn #field_name (self, _: #repeated_fields_error_type_name) -> #builder_name <#( #target_generics ),*> {
+                pub fn #method_name (self, _: #repeated_fields_error_type_name) -> #builder_name <#( #target_generics ),*> {
                     self
                 }
             }
         })
     }
 
+    fn method_name(field: &FieldInfo, field_name: &Ident) -> Result<Ident, Error> {
```

Comment:
Please change it to `setter_method_name`. While as a variable `method_name` makes sense inside the context of `field_impl`, it is pretty vague as a struct level method name. Especially when it's under `StructInfo`.

Speaking of which - why is this not under `FieldInfo`? And `FieldInfo` already has it's `name` inside it - there is no need to pass it twice.

Reply:
Yes, that makes sense. Changed in cc1c5da